test/kitchen/site-cookbooks/dd-system-probe-check/recipes/default.rb

```diff
@@ -38,7 +38,7 @@
       fi
 
       if [[ ${dev_name} =~ ^/dev/mapper ]]; then
-         lvresize -L +10G ${dev_name}
+         lvresize -L +20G ${dev_name}
```

Comment:
I'd hold on this change until #15670 merges, and then this code is no longer necessary.

Reply:
Thanks